sw/device/tests/sim_dv/ast_usb_clk_calib.c

```diff
@@ -27,8 +28,7 @@ static uint32_t cast_safely(uint64_t val) {
 
 static uint32_t device_usb_count;
 static uint32_t aon_clk_period_us;
-const static uint32_t kSoFPeriodUs = 1000;
-const static uint32_t kNumSoF = 2;
+static volatile const uint32_t kSoFPeriodUs = 1000;
```

Comment:
nit: would you like to add a comment explaining why this needs to be `volatile`?

Reply:
o yeah good point, let me do that.